### PR TITLE
Docs: Only Darwin for PNLF

### PR DESCRIPTION
**Pull request by Lorys89**
add _sta method return for win acpi

### CODE REVIEW

**Comment by Lorys89** [comment]
IMG_20201126_003434

**Comment by Lorys89** [comment]
putting method _sta with darwin, on windows there is no more error.

**Comment by 1alessandro1** [comment]
I still don't get why you use OpenCore to boot Windows. That's just dumb.

Use the boot menu and separate disks...

**Comment by Lorys89** [comment]
> Continuo a non capire perché usi OpenCore per avviare Windows. È solo stupido.
> 
> Usa il menu di avvio e dischi separati ...

and a bootloader that has dual boot, why should I use boot windows from bios? if adding the method _STA resolves? 
it makes no sense what you say.

**Comment by 1alessandro1** [comment]
Contional OS patching has been demonstrated to be unreliable, that's why OpenCore does not have policies about when it's necessary to apply the settings described in the config.plist or not.

Just does.

So why are you applying random _STA here and there if booting from the UEFI boot menu is a much cleaner approach to keep OSes and their relative firmwares/drivers separate?

**Comment by Lorys89** [comment]
with your statement, you consider useless the methods is already present in ssdt ec, pmc etc etc
I do not share it, but I accept your thoughts. I await the opinion of the devs.

**Comment by vit9696** [comment]
Makes sense to me, we did not have that on our devices. Could you include the same change for CFL table nearby?

**Comment by Lorys89** [comment]
> Makes sense to me, we did not have that on our devices. Could you include the same change for CFL table nearby?

do you want to apply change to ssdt pnlf and ssdt pnlf cfl? in both?

**Comment by vit9696** [comment]
Correct.

**Comment by Lorys89** [comment]
> Correct.

ok, also applied to ssdt pnlf

File: Manual/SSDT-PNLF.dsl [hunk 1]
```diff
@@ -46,7 +46,17 @@ DefinitionBlock("", "SSDT", 2, "ACDT", "PNLF", 0)
         // 19: CoffeeLake 0xffff
         // 99: Other (requires custom AppleBacklightInjector.kext/WhateverGreen.kext)
         Name(_UID, 0)
-        Name(_STA, 0x0B)
+        Method (_STA, 0, NotSerialized)
+            {
```
Comment (stevezhengshiqi) [inline Manual/SSDT-PNLF.dsl:50]: Thank you for this. Would you mind adjusting the indentation here?

**Review by Lorys89 — COMMENTED** [review]
fix identation

**Comment by Lorys89** [comment]
> Correct.

do I need to do anything else?

**Comment by vit9696** [comment]
The change looks good to me. Will merge when we get CI fixed.

**Comment by Lorys89** [comment]
> Il cambiamento mi sembra buono. Si fonderanno quando avremo riparato CI.

ok thank you,, but I have ice lake laptop, and the igpu doesn't work well, I can't patch hdmi, and problems with backlight and maximum bkl value, depends on weg, after 3 minutes that start, the backlight turns on, depends on weg.

**Comment by idalin** [comment]
> > Il cambiamento mi sembra buono. Si fonderanno quando avremo riparato CI.
> 
> ok thank you,, but I have ice lake laptop, and the igpu doesn't work well, I can't patch hdmi, and problems with backlight and maximum bkl value, depends on weg, after 3 minutes that start, the backlight turns on, depends on weg.

https://github.com/acidanthera/WhateverGreen/pull/76   try this.

**Comment by Lorys89** [comment]
> > > Il cambiamento mi sembra buono. Si fonderanno quando avremo riparato CI.
> > 
> > 
> > ok thank you,, but I have ice lake laptop, and the igpu doesn't work well, I can't patch hdmi, and problems with backlight and maximum bkl value, depends on weg, after 3 minutes that start, the backlight turns on, depends on weg.
> 
> #76 try this.

thank you, later I fill in and try 😘

**Comment by Lorys89** [comment]
> > > Il cambiamento mi sembra buono. Si fonderanno quando avremo riparato CI.
> > 
> > 
> > ok thank you,, but I have ice lake laptop, and the igpu doesn't work well, I can't patch hdmi, and problems with backlight and maximum bkl value, depends on weg, after 3 minutes that start, the backlight turns on, depends on weg.
> 
> #76 try this.

thank you so much, work well backlight right away, I don't have to wait 3 minutes, great pr your 💪

**Pull request merged**